src/main/java/com/google/sps/servlets/QuizPoints.java

```diff
@@ -1,5 +1,5 @@
 package com.google.sps.servlets;
-
```

Comment:
Please remove these extra spaces that get added to files. I have no idea where they keep coming from (because I highly doubt you guys are doing this intentionally) but it might be worth trying to figure out the source of them so we can prevent them from sneaking into future PRs.

Reply:
Got it!